tasks/broad/GermlineVariantDiscovery.wdl

```diff
@@ -174,6 +174,44 @@ task MergeVCFs {
   }
 }
 
+task Reblock {
+
+  input {
+    File gvcf
+    File gvcf_index
+    File ref_dict
+    File ref_fasta
+    File ref_fasta_index
+    String output_vcf_filename
+    String docker_image = "us.gcr.io/broad-dsde-methods/updated_reblocking@sha256:b3f98210cb5156cfe924ad017f4b834a5367b33281f5185b1a61f9e5387ed16f"
+  }
+
+  Int disk_size = ceil(size(gvcf, "GiB")) * 2 + 3
+
+  command {
+    gatk --java-options "-Xms3g -Xmx3g" \
+      ReblockGVCF \
+      -R ~{ref_fasta} \
+      -V ~{gvcf} \
+      -do-qual-approx \
+      --floor-blocks -GQB 20 -GQB 30 -GQB 40 \
```

Comment:
Can you explain that you've changed the --floor-blocks and removed the drop-low-quals parameter?  (I assume this is due to changes in the gatk code, but should be in the changelog)

Reply:
--drop-low-quals made holes in the GVCF, which weren't worth the trouble for the space savings we thought we were getting.  The changelog does say no gaps -- does it matter how?

--floor-blocks quantizes the GQ values by rounding down to the floor of the bin the GQ belongs to, which improves compression significantly.  That should be in the changelog.